src/Pagination.jsx

```diff
@@ -347,7 +347,11 @@ export default class Pagination extends React.Component {
               onChange={this.handleKeyUp}
               size="3"
             />
-            <span className={`${prefixCls}-slash`}>／</span>
+            <span className={`${prefixCls}-divider`} >
```

Comment:
I don't think use itemRender for this feature is a good idea. Maybe just add a prop `dividerSymbol` for it.